src/presentational-components/shared/table-toolbar-view.js

```diff
@@ -131,7 +116,7 @@ export const TableToolbarView = ({
           titleSingular={titleSingular}
           filterValue={filterValue}
           setFilterValue={setFilterValue}
-          sortBy={sortBy}
+          sortBy={orderBy}
```

Comment:
Maybe we can rename it back to `sortBy`

Reply:
I would not do it because of mixing up two variables with the same name

Reply:
@fhlavac there already is a prod called `orderBy`. We are out of luck :(